salt/orchestrate/logging/init.sls

```diff
@@ -39,6 +39,49 @@ build_logging_nodes:
     - tgt_type: compound
     - highstate: True
 
+# Obtain the grains for one of the elasticsearch nodes
+{% set grains = salt.saltutil.runner(
+    'mine.get',
+    tgt='roles:elasticsearch', fun='grains.item', tgt_type='grain'
+    ).items()[0][1] %}
+# PUT the mapper template into the ES _template index
+put_elasticsearch_mapper_template:
+  http.query:
+    - name: http://{{ grains['ec2:local_ipv4'] }}:9200/_template/logstash
+    - data: '
```

Comment:
Just a nit, but you could use the `|` operator instead of quoting the whole string.